xmtp/src/client.rs

```diff
@@ -105,13 +105,22 @@ where
 
         let mut contacts = vec![];
         for envelope in response.envelopes {
-            let contact_bundle = Contact::from_bytes(envelope.message, wallet_address.to_string())?;
+            let contact_bundle = Contact::from_bytes(envelope.message)?;
             contacts.push(contact_bundle);
         }
 
         Ok(contacts)
     }
 
+    pub async fn my_other_devices(&self) -> Result<Vec<Contact>, ClientError> {
```

Comment:
This snuck in from another branch, but it's useful and small, so I'll leave it here in case someone else needs it.